libs/watt/package/menu/watt-menu-group.component.ts

```diff
@@ -0,0 +1,63 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { Component, ViewEncapsulation, ChangeDetectionStrategy, input } from '@angular/core';
+
+@Component({
+  selector: 'watt-menu-group',
+  template: `
+    <div class="watt-menu-group">
+      @if (label()) {
+        <div class="watt-menu-group-heading watt-text-s-highlighted">{{ label() }}</div>
+      }
+      <ng-content />
+    </div>
```

Comment:
question: this `div` has the same class as the "host" component on line 55. Is this intentional? Can it cause some issues? Is there a need for an extra `div.watt-menu-group`?